ztp/extra-manifests-builder/06-kdump/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+GOPATH=${GOPATH:-${HOME}/go}
+GOBIN=${GOBIN:-${GOPATH}/bin}
+MCMAKER=${MCMAKER:-${GOBIN}/mcmaker}
+MCPROLE=${MCPROLE:-master}
+
+${MCMAKER} -stdout -name 06-kdump-enable -mcp ${MCPROLE} \
+	unit -source kdump.service -name kdump.service -enable=true
```

Comment:
Looks like I should update mcmaker so it can just enable a unit without having to supply the source of it...

Reply:
Yes, all we need is to enable the service. Not include anything